storage/src/vespa/storage/storageserver/changedbucketownershiphandler.cpp

```diff
@@ -451,4 +474,10 @@ ChangedBucketOwnershipHandler::onInternalReply(
     return (reply->getType() == AbortBucketOperationsReply::ID);
 }
 
+void
+ChangedBucketOwnershipHandler::onClose()
+{
+    _state_sync_executor.shutdown();
```

Comment:
should do sync somewhere, either here (to wait for pending tasks) or in the destructor of ChangedBucketOwnershipHandler

Reply:
Added an explicit `sync()` after `shutdown()`